README.md

```diff
@@ -176,6 +192,28 @@ Your should see report file generated as ExtentReport.html under the target fold
 ##WIP
 iOS WebTest using Safari
 
+##FAQ
+Q. Is this framework supports to run multiple IOS simulators?
+
+A. No, Apple does not allow to do so.
```

Comment:
Yes, using webdriveragent this is possible. But i guess with appium-webdriveragent is still in beta release. So we can say multiple emulator will be supported in appium's future releases.